Node/src/TeamsChatConnector.ts

```diff
@@ -159,7 +164,32 @@ export class TeamsChatConnector extends builder.ChatConnector {
       }
     }
     if (realEvents.length > 0) {
-      super.onDispatchEvents(realEvents, callback);
+      // Filter out @mention
```

Comment:
I think you should only filter out at-mentions of the bot. In a command like "<at>Trello</at> assign to <at>Adrian</at> and <at>Dan</at>", the bot mention carries no semantic meaning (it's just for routing to the bot) but the people mentions do and could conceivably be relevant to intent.